_posts/2020-10-31-manage-jdks-with-sdkman.md

```diff
@@ -0,0 +1,180 @@
+Available Java Versions
+================================================================================
+ Vendor        | Use | Version      | Dist    | Status     | Identifier
+--------------------------------------------------------------------------------
+ AdoptOpenJDK  |     | 15.0.1.j9    | adpt    |            | 15.0.1.j9-adpt
+               |     | 15.0.1.hs    | adpt    |            | 15.0.1.hs-adpt
+               |     | 13.0.2.j9    | adpt    |            | 13.0.2.j9-adpt
+               |     | 13.0.2.hs    | adpt    |            | 13.0.2.hs-adpt
+               |     | 12.0.2.j9    | adpt    |            | 12.0.2.j9-adpt
+               |     | 12.0.2.hs    | adpt    |            | 12.0.2.hs-adpt
+               |     | 11.0.9.open  | adpt    |            | 11.0.9.open-adpt
+               |     | 11.0.9.j9    | adpt    |            | 11.0.9.j9-adpt
+               | >>> | 11.0.9.hs    | adpt    | installed  | 11.0.9.hs-adpt
+               |     | 8.0.272.j9   | adpt    |            | 8.0.272.j9-adpt
+               |     | 8.0.272.hs   | adpt    |            | 8.0.272.hs-adpt
+ Alibaba       |     | 11.0.8       | albba   |            | 11.0.8-albba
+               |     | 8u262        | albba   |            | 8u262-albba
+ Amazon        |     | 15.0.1       | amzn    |            | 15.0.1-amzn
+               |     | 11.0.9       | amzn    |            | 11.0.9-amzn
+               |     | 8.0.272      | amzn    |            | 8.0.272-amzn
+================================================================================
+```
+
+Now to install the JDK of our choice run: `sdk install java <candidate>`. For example: `sdk install java 15.0.1.j9-adpt`
+
+SDKMAN! will now download the desired JDK and will ask us if we want to set it as default.
+```
+Downloading: java 15.0.1.j9-adpt
+
+In progress...
+
+Do you want java 15.0.1.j9-adpt to be set as default? (Y/n):
+```
+
+If we run now again `sdk list java` we should be able to see which versions we have installed:
+```text
+================================================================================
+Available Java Versions
+================================================================================
+ Vendor        | Use | Version      | Dist    | Status     | Identifier
+--------------------------------------------------------------------------------
+ AdoptOpenJDK  | >>> | 15.0.1.j9    | adpt    |  installed | 15.0.1.j9-adpt
+```
+
+## Setting the Global JDK
+
+With the 6-month version JDK cycle that is now being released, we might want to add a global (default) JDK for our computer that is sensible for example an LTS version.
+
+To do so run: `sdk default java <canditate>`. For example: `sdk default java 11.0.9.hs-adpt`.
+
+```text
+Default java version set to 11.0.9.hs-adpt
+```
+
+## Setting the Local JDK
+
+Sometimes, we might want to try out the new Java version, but not set it globally.
```

Comment:
Please add a sentence about what a "local JDK" means. What's the difference to a "per project" JDK? I guess it means we're using the local JDK only in the current shell, but I'm guessing.

Reply:
I did add the requested sentence :)